### PR TITLE
Traversal scrape code

### PR DESCRIPTION
**Pull request by bmcorser**
A script that traverses from a company to its contacts and interactions, saving to the odata database and sending to leeloo.

### CODE REVIEW

**Comment by uktitools** [comment]
Can one of the admins verify this patch?

File: resync.py [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+
+def send_forever(django_tablename, django_dict):
+    sent = False
+    while not sent:
+        try:
+            response = leeloo.send(django_tablename, [django_dict])[0]
+            sent = True
+            return response
+        except Exception:
+            pass
```
Comment (canni) [inline resync.py]: Sentry?

File: resync.py [hunk 2]
```diff
@@ -0,0 +1,86 @@
...
+
+
+def sync(odata_tablename, odata_pkey, django_tablename, failure_fmt):
+    print("{0} -> {1}".format(odata_tablename, django_tablename))
```
Comment (canni) [inline resync.py]: Prints aren't portable, use `logging` ;)

File: resync.py [hunk 3]
```diff
@@ -0,0 +1,86 @@
...
+            select = sqla.select([django_table]).where(comparitor)
+            result = django_metadata.bind.execute(select).fetchone()
+            if result is not None:
+                skipped = skipped + 1
+                continue
+            django_dict = {
+                k: str(v) for k, v in
+                transform.odata_to_django(odata_tablename, dict(odata_row)).items()
+            }
+            response = send_forever(django_tablename, django_dict)
```
Comment (canni) [inline resync.py]: Can we have thread worker pool for that, single failing send will block the whole thing?

File: resync.py [hunk 4]
```diff
@@ -0,0 +1,86 @@
...
+            'ActivityId',
+            'company_interaction',
+            'interaction-failures/{0}'
+        ),
+    )
+    for entity_spec in entity_specs:
+        sync(*entity_spec)
+
+if __name__ == '__main__':
+    main()
```
Comment (canni) [inline resync.py]: A `pyramid` console script would handle logging & be import path independent, but for now it's good enough :D

**Pull request merged**